dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ApplicationConfig.java

```diff
@@ -267,4 +270,13 @@ public void setShutwait(String shutwait) {
         System.setProperty( Constants.SHUTDOWN_WAIT_KEY, shutwait);
         this.shutwait = shutwait;
     }
+
+    public boolean isEnableTelnet() {

```

Comment:
I don't think this is a application config.
Maybe protocol config is better because this config is only useful for dubbo protocol.

Reply:
you are right